introduction.tex

```diff
@@ -55,15 +55,19 @@ \subsection{Scenario Descriptions}
 provides natural uranium, which is enriched by an 'enrichment'
 facility to \gls{UOX}, while disposing enrichment waste (tailings)
 to the 'sink' facility. The enriched \gls{UOX} is used
-in the \gls{LWR}s and \gls{UOX} waste is produced. Then the waste
-would either go directly to the sink (scenario 1), or is reprocessed
-for further use (scenario 2 and 3). In scenario 2, the waste is 
-separated in the separations facility (reprocessing plant), into
-two streams, Plutonium and Uranium. The plutonium is mixed with depleted uranium from enrichment
-to create \gls{MOX}, and the reprocessed uranium is stockpiled in the fabrication facility.
-The produced \gls{MOX} is then used in French \gls{PWR}s.
-In scenario 3, the waste goes through the same process but is mixed
-to create \gls{SFR} fuel.
+in the \gls{LWR}s and \gls{UOX} waste is produced. Then the spent fuel
+goes directly to the sink. However, in the case of France, the spent fuel
+is reprocessed to be fabricated into \gls{MOX}.
+
+
+This paper explores two scenarios.
+First is the case of having regional repositories. Taking into consideration 
+the spent fuel inventory, repository research progress and geographical location,
+regional repositories of reasonable capacity ($~70,000$ MTHM) are considered.
+Second is to consider French-centralized repository, where France accepts all
+spent fuel from EU nations. It is also analyzed how much \gls{MOX} can be produced
+from the spent fuel received. In essence, the paper explores how taking fuel
+from EU nations can have an economic incentive to France.
```

Comment:
"have an economic incentive to" doesn't make sense. 

Perhaps: "may be incentivized by French pursuit of an advanced reactor fleet."
OR: "may be economically beneficial to France."
OR: "may satisfy the MOX demands of the French reactor fleet."